cmake/externals/projects_modules/ITK.cmake

```diff
@@ -84,7 +84,7 @@ set(cmake_args
 ## Check if patch has to be applied
 ## #############################################################################
 
-ep_GeneratePatchCommand(ITK ITK_PATCH_COMMAND ITK_Mac_Rpath.patch)
+#ep_GeneratePatchCommand(ITK ITK_PATCH_COMMAND ITK_Mac_Rpath.patch)
```

Comment:
Maybe we could remove this line.
But if you prefer, we could keep it in comments.

Reply:
Done :)